### DISCUSSION TITLE
How do you edit the long URL behind a slug?

### DISCUSSION THREAD

**Discussion by RyanHakurei** [shlinkio/shlink#2162] [Help wanted] [answered]
### Shlink version

4.1.1

### PHP version

???

### How do you serve Shlink

Docker image

### Database engine

MariaDB

### Database version

???

### Summary

I see that the ability to edit long URLs was added in [688](https://github.com/shlinkio/shlink/pull/688), but from what I can see there is no way to actually edit the long URL behind a slug? I would expect there to be a short-url:edit command but that does not exist.

**Comment by acelaya** [comment, marked as answer]
This looks like an overlook.

You can edit short URLs via API, which is what most of the people use https://api-spec.shlink.io/#/Short%20URLs/editShortUrl

The command line is provided as a convenient way to interact with your instance, and the intention is that there is API<->CLI parity, but it's not the first time some features are missing by mistake.

I'll add the missing command.

EDIT: The command is now implemented and will be released with Shlink 4.2

**Reply by RyanHakurei** [reply]
Cool, no rush I was just wondering because I checked and found that it was a feature and was simply confused as to why I could not find it.

**Reply by acelaya** [reply]
Feel free to track progress here https://github.com/shlinkio/shlink/issues/2163
